### PR TITLE
Send inviatation code only to the host.

### PR DESCRIPTION
**Pull request by hilalcomak**

### CODE REVIEW

**Review by apple00juice — APPROVED** [review]
Looks good except the token in linkRegisteredUser, as mentioned in the other pull request

File: src/main/java/ch/uzh/ifi/hase/soprafs22/rest/mapper/DTOMapper.java [hunk 1]
```diff
@@ -34,7 +34,14 @@ public abstract class DTOMapper {
     @Mapping(source = "team", target = "team")
     public abstract PlayerGetDTO convertIPlayerToPlayerGetDTO(IPlayer player);
 
-
+    //add invitation code if token matches
+    public LobbyGetDTO convertILobbyToLobbyGetDTO(ILobby lobby, String token) {
+        LobbyGetDTO ret = convertILobbyToLobbyGetDTO(lobby);
```
Comment (paolovic) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/rest/mapper/DTOMapper.java:39]: can you think of a more descriptive name than "ret"?

**Review by paolovic — APPROVED** [review]
great, only 1 naming i would do differently

File: src/main/java/ch/uzh/ifi/hase/soprafs22/rest/mapper/DTOMapper.java (hunk 1 above)
Comment (hilalcomak) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/rest/mapper/DTOMapper.java]: sure, done.

**Pull request merged**